Makefile.core.mk

```diff
@@ -703,6 +703,20 @@ show.goenv: ; $(info $(H) go environment...)
 show.%: ; $(info $* $(H) $($*))
 	$(Q) true
 
+#-----------------------------------------------------------------------------
+# Target: custom resource definitions
+#-----------------------------------------------------------------------------
+
+API_TMP := $(shell mktemp -d -u)/api
```

Comment:
can we do this in the target so we don't make a directory every time we call make?

Reply:
This command actually does not create any temp directory. It just outputs a temp directory name. Moved this into the target anyways.